install/install.sh

```diff
@@ -18,15 +18,19 @@ echo ""
 echo "* Updating package list        "
 apt-get update >> "$install_log" 2>&1
 echo "* Installing nginx, python & pip      "
-apt-get -q -y install dnsutils nginx python python-dev python-pip >> "$install_log" 2>&1
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    apt-get -q -y install dnsutils nginx python build-essential libffi-dev libssl-dev python-dev python-pip >> "$install_log" 2>&1
+else
+    apt-get -q -y install dnsutils nginx python python-dev python-pip >> "$install_log" 2>&1
+fi
 if [ ! -f /etc/init.d/mysql* ]; then
     echo "* Installing MySQL (root password will be empty!)"
     DEBIAN_FRONTEND=noninteractive apt-get install -y mysql-server >> "$install_log" 2>&1
 fi
 echo "* Update setuptools            "
-easy_install -U setuptools >> "$install_log" 2>&1
+pip install --upgrade setuptools  >> "$install_log" 2>&1
 echo "* Installing pip dependencies"
-pip install twisted pyopenssl flask-sqlalchemy flask passlib pymysql service_identity pycrypto flask-wtf netifaces gunicorn >> "$install_log" 2>&1
+pip install werkzeug ipaddress enum34 cryptography idna sqlalchemy twisted pyopenssl flask-sqlalchemy flask passlib pymysql service_identity pycrypto flask-wtf netifaces gunicorn >> "$install_log" 2>&1
```

Comment:
werkzeug should be inferred by installing flask... Is this no longer the case?

Reply:
remove werkzeug in the latest commit.

Reply:
It's still in here :)